var/spack/repos/builtin/packages/rocm-validation-suite/package.py

```diff
@@ -183,10 +183,14 @@ def patch(self):
             filter_file(
                 "@ROCM_PATH@/rvs", self.spec.prefix.rvs, "rvs/conf/deviceid.sh.in", string=True
             )
-        elif "@5.2.0:" in self.spec:
+        elif "@5.2.0:5.4" in self.spec:
             filter_file(
                 "@ROCM_PATH@/bin", self.spec.prefix.bin, "rvs/conf/deviceid.sh.in", string=True
             )
+        elif "@5.5:" in self.spec:
```

Comment:
Same here for use of `self.spec.satisfies()`.

Reply:
Updated as recommended